### PR TITLE
Update docs for fields allowed at root of the CRD schema

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/kubernetes/pull/65357

The list of fields can be found here: https://github.com/kubernetes/kubernetes/blob/abdfd7cebf6dde5c73abd259138e7ecd0088488b/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go#L537

This is for 1.12.

Preview: https://5b7c23e9b13fb126244fa496--kubernetes-io-vnext-staging.netlify.com/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/

/sig api-machinery
/assign sttts

### CODE REVIEW

File: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md [hunk 1]
```diff
@@ -469,7 +470,28 @@ When the status subresource is enabled, the `/status` subresource for the custom
 - `PUT` requests to the `/status` subresource only validate the status stanza of the custom resource.
 - `PUT`/`POST`/`PATCH` requests to the custom resource ignore changes to the status stanza.
 - Any changes to the spec stanza increments the value at `.metadata.generation`.
-- `properties`, `required` and `description` are the only constructs allowed in the root of the CRD OpenAPI validation schema.
+- Only the following constructs allowed in the root of the CRD OpenAPI validation schema:
```
Comment (sttts) [inline content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md]: "only" :)

**Comment by nikhita** [comment]
/assign @tengqm
for approval

File: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md (hunk 1 above)
Comment (Bradamant3) [reply content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md]: -> Only the following constructs are allowed at the root of the CRD OpenAPI validation schema:

(fix fragment and preposition)

Comment (nikhita) [reply content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md]: Done.

**Comment by chenopis** [comment]
@Bradamant3 Your feedback was addressed.
/lgtm
/approve

**Pull request merged**